cert/file.go

```diff
@@ -126,25 +128,31 @@ func (f *File) parse() (err error) {
 	return nil
 }
 
-// Set ensures the file has the right owner/group and mode.
-func (f *File) setPermissions() error {
+// CheckPermissions checks that the permissions on disk match what we intend.  If the
+// file doesn't exist, then the raw error for stat is returned.
+func (f *File) CheckPermissions() error {
```

Comment:
this replaces the `setPermissions` method which was unused (I forgot to clean it out when I added atomic renames).